src/core/ts/shared-resources/models/BaseTranslationState.ts

```diff
@@ -18,7 +18,6 @@ export enum TranslationStatus {
   SOURCE_LANGUAGE_UNDERSTOOD = "SOURCE_LANGUAGE_UNDERSTOOD",
   TRANSLATION_UNSUPPORTED = "TRANSLATION_UNSUPPORTED",
   OFFER = "OFFER",
-  DOWNLOADING_TRANSLATION_MODEL = "DOWNLOADING_TRANSLATION_MODEL",
```

Comment:
@motin Does it mean that the UI will not show that models are being downloaded?

Reply:
Good question but no, this is a cleanup of a previously planned translation status that never materialized. Download progress is communicated during the TRANSLATED status using other attributes than the translation status attribute.